dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -144,10 +144,10 @@ private static void printStaticLinkageReport(StaticLinkageCheckReport report) {
     ImmutableList.Builder<Path> jarFileBuilder = ImmutableList.builder();
 
     // TODO(suztomo): add logic to convert Maven BOM to list of Maven coordinates as per README.md
-    if (!linkageCheckOption.getJarFileList().isEmpty()) {
-      jarFileBuilder.addAll(linkageCheckOption.getJarFileList());
-    } else if (!linkageCheckOption.getMavenCoordinatesList().isEmpty()) {
-      for (String mavenCoordinates : linkageCheckOption.getMavenCoordinatesList()) {
+    if (!linkageCheckOption.getJarFiles().isEmpty()) {
```

Comment:
You don't have to bother to check for empty. `addAll(empty)` is a no-op, and looping over an empty collection is also a no-op.

Reply:
Removed unnecessary if-statements.